lumicks/pylake/kymotracker/kymotracker.py

```diff
@@ -339,26 +348,60 @@ def refine_lines_centroid(lines, line_width):
         # Refinement only does something when line_width in pixels is larger than 1
         raise ValueError("line_width may not be smaller than 1")
 
-    interpolated_lines = [line.interpolate() for line in lines]
-    time_idx = np.round(np.array(np.hstack([line.time_idx for line in interpolated_lines]))).astype(
-        int
-    )
+    # convert line_width (pixel units) to physical units expected by refine_tracks_centroid
+    track_width = line_width * lines[0]._kymo.pixelsize[0]
+    return refine_tracks_centroid(lines, track_width)
+
+
+def refine_tracks_centroid(tracks, track_width=None):
+    """Refine the tracks based on the brightness-weighted centroid.
+
+    This function interpolates the determined tracks (in time) and then uses the pixels in the vicinity of the
+    tracks to make small adjustments to the estimated location. The refinement correction is
+    computed by considering the brightness weighted centroid.
+
+    Parameters
+    ----------
+    tracks : List[pylake.KymoTrack]
+        Detected tracks on a kymograph
+    track_width : float
+        Expected spatial track width in physical units. Must be larger than zero.
+        If `None`, the default is 0.35 (half the wavelength of the red limit of the visible spectrum)
+        for kymographs calibrated in microns and 1 for kymographs calibrated in kilobase pairs (based
+        on 0.34 nm/bp for duplex DNA)
```

Comment:
I know we discussed this two days ago, but the double default did get me thinking, it does seem a bit odd that we use a different default depending on how the `Kymograph` is calibrated. For pixels, I totally get it, because you have no physical bearing, but for `kbp`, maybe it would make sense to have the default for kbp be `0.34/0.35`? Since that's what you actually expect for the line width, no?

Reply:
I think the documentation should also clearly mention the rounding strategy used when determining how many pixels to use (`ceil`).

Reply:
for the first one, I hardcoded in the value because apparently I forgot that in python you can put math in your code 😒 

For the second comment, this sounds ok but I'll do it in a separate PR since this one is already getting a bit crazy

Reply:
Right. Makes sense. 👍